app/assets/stylesheets/components/_markdown-editor.scss

```diff
@@ -83,24 +82,27 @@
 
 .app-c-markdown-editor__toolbar {
```

Comment:
Non-blocker: an idea to get `.app-c-markdown-editor__toolbar` aligned would be set it to `display: table`. This will also remove the spacing between buttons.

Reply:
Looks good to be merged to me